tests/e2e/tidbcluster/serial.go

```diff
@@ -690,6 +694,95 @@ var _ = ginkgo.Describe("[Serial]", func() {
 			})
 		})
 
+		ginkgo.Context("From prev major version", func() {
+
+			ginkgo.BeforeEach(func() {
+				operatorVersion = OperatorPrevMajorVersion
+			})
+
+			ginkgo.It("should not trigger rolling-update and database work fine", func() {
```

Comment:
`We need to cover the TLS case.`, https://github.com/pingcap/tidb-operator/issues/4441#issuecomment-1053807798

Reply:
Done